components/Button/Button.styles.js

```diff
@@ -0,0 +1,15 @@
+import styled, { css } from 'styled-components';
+import { pxToRem } from '../_theme';
+
+const StyledButton = styled.button`
+  background: linear-gradient(to left,${props => props.theme.variables.colors.primary.default} 50%,${props => props.theme.variables.colors.secondary.default});
```

Comment:
Instead of adding all these inline functions, which can be difficult to follow, try wrapping all the CSS in a single function with some destructuring. You could go a step further and destructure theme for `variables: { colors }` & `borderRadius`

```
${({ theme, attributes }) => css`
    background: linear-gradient(
      to left,
      ${theme.variables.colors.primary.default} 50%,
      ${theme.variables.colors.secondary.default}
    );
    border-radius: ${theme.borderRadius.small};
    color: ${theme.variables.colors.grayScale.white};
    display: inline-block;
    font-size: ${pxToRem(16)};
    height: auto;
    margin-right: ${pxToRem(8)};
    width: ${attributes.width ? pxToRem(attributes.width) : pxToRem(50)};
  `}
```

As an example https://github.com/wsfuller/next-github-viewer/blob/issue-33-hero-component/components/Hero/Hero.styles.js

Like to go back through and clean up some of the other styles so they all fall in line with this style